development/compiling/compiling_for_android.rst

```diff
@@ -8,147 +8,154 @@ Compiling for Android
 
 ::
 
-    C:\godot> scons platform=android target=release
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
+    scons platform=android target=release
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux
+    ./gradlew build
 
-(on Linux or macOS, execute the `gradlew` script with `./gradlew build`)
 
-The resulting APK is in:
-
-::
+The resulting APK will be located at ``bin/android_release.apk``.
 
-    bin\android_release.apk
-
--  Debug template (used when exporting with "Debugging Enabled" ON)
+-  Debug template (used when exporting with **Debugging Enabled** checked)
 
 ::
 
-    C:\godot> scons platform=android target=release_debug
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
-
-The resulting APK is in:
+    scons platform=android target=release_debug
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux/macOS
+    ./gradlew build
 
-::
 
-    bin\android_debug.apk
+The resulting APK will be located at ``bin/android_debug.apk``.
 
 Faster compilation
 ~~~~~~~~~~~~~~~~~~
 
-If you are on Unix or installed PyWin32 on Windows and have multiple CPU
-cores available, you can speed up the compilation by adding the ``-jX``
-argument to the SCons command, where ``X`` is the number of cores that you
-want to allocate to the compilation, e.g. ``scons -j4``.
-
+If you have multiple CPU cores available, you can speed up compilation
+by adding the ``-jX`` argument to the SCons command, where ``X`` is the
+number of cores you want to allocate to building, e.g. ``scons -j4``.
 
 Adding support for x86 devices
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you also want to include support for x86 devices, run the scons command
+If you also want to include support for x86 devices, run the SCons command
 a second time with the ``android_arch=x86`` argument before building the APK
-with Gradle. For example for the release template:
+with Gradle. For example, for the release template:
 
 ::
 
-    C:\godot> scons platform=android target=release
-    C:\godot> scons platform=android target=release android_arch=x86
-    C:\godot> cd platform/android/java
-    C:\godot\platform\android\java> gradlew build
+    scons platform=android target=release
+    scons platform=android target=release android_arch=x86
+    cd platform/android/java
+    # On Windows
+    .\gradlew build
+    # On Linux/macOS
+    ./gradlew build
 
 
-This will create a fat binary that works in both platforms, but will add
+This will create a fat binary that works on both platforms, but will add
 about 6 megabytes to the APK.
 
 Troubleshooting
 ~~~~~~~~~~~~~~~
 
 It might be necessary to clean the build cache between two APK compilations,
 as some users have reported issues when building the two export templates
-one after the other.
+one after the other. This can be done using the following command:
+
+::
+
+    scons platform=android -c
```

Comment:
This will only clean build objects for the default arch (`armv7`). One should specify the exact same arguments as used for compilation to `scons -c`.

This paragraph was added in 2016 and honestly I never saw any mention of it being needed (and as we well know most people don't read docs, or wouldn't know how to clean things here, so if it was indeed needed we'd have heard about it by now), so I'd say best remove it.